pro/src/components/pages/Offerers/Offerer/VenueV1/VenueEdition/ButtonImageDelete/ButtonImageDelete.tsx

```diff
@@ -20,15 +21,24 @@ export const ButtonImageDelete = ({
 }: ButtonImageDeleteProps): JSX.Element => {
   const { visible, showModal, hideModal } = useModal()
   const [isLoading, setIsLoading] = useState(false)
+  const notification = useNotification()
 
   const tryToDeleteVenueImage = useCallback(async () => {
     setIsLoading(true)
-    await deleteVenueImage({ venueId })
-
-    onDeleteImage()
-    setIsLoading(false)
-    hideModal()
-  }, [onDeleteImage, hideModal, venueId])
+    try {
+      await deleteVenueImage({ venueId })
+
+      onDeleteImage()
+      hideModal()
+    } catch (exception) {
+      hideModal()
+      notification.error(
+        'Une erreur est survenue. Merci de réessayer plus tard.'
+      )
+    } finally {
+      setIsLoading(false)
+    }
```

Comment:
Je pense ça serait mieux si on met ici un truc du genre :
```suggestion
    deleteVenueImage({ venueId })
      .then(onDeleteImage)
      .catch(() =>
        notification.error(
          'Une erreur est survenue. Merci de réessayer plus tard.'
        )
      )
      .finally(() => {
        hideModal()
        setIsLoading(false)
      })
```
Ou sinon comme Romain l'a suggéré dans #1629